metrics/pcp/pcp.go

```diff
@@ -0,0 +1,129 @@
+package pcp
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/performancecopilot/speed"
+)
+
+// Reporter encapsulates a speed client
+type Reporter struct {
+	c *speed.PCPClient
+}
+
+// NewReporter creates a new reporter instance
```

Comment:
1. Please use proper punctuation, as above.
2. Please describe a bit the meaning of the appname parameter, and the behavior if there is some error or misconfiguration.